ci/requirements-3.5-dev.yml

```diff
@@ -25,7 +25,7 @@ dependencies:
   - pre_commit
   - psycopg2
   - pyarrow<0.12
-  - pymapd>=0.8.3,<0.11.0
+  - pymapd>=0.12
```

Comment:
I think this needs to remain unchanged, because one of the transitive dependencies of `pymapd>=0.12` is numpy 1.16.3 or greater and there's no Python 3.5 on conda-forge for that.

If sessions aren't available in older pymapd versions then we should raise an error if a user is using 3.5 and they pass in a non-`None` `sessionid` argument.

Reply:
Sounds good.